### PR TITLE
Capitalise Words in Name and Other fixes

### LINKED ISSUE #121: Make Name of Member capitalised and allow lower-case input
Essentially if we have a new member of name "john doe", it will become  "John Doe" and stored as that

### PR DESCRIPTION
**Pull request by wr1159**
Closes #121 
* Updated CapitaliseWordsInString to capitalise all words in string (for example: john doe sain -> John Doe Sain)
  * Current iteration also turns the rest of the word into lower case (for example: JOHN DOE -> John Doe)
* Run CapitaliseWordsInString when initialising Name to convert into Pascal Case with space
* Fix gender being displayed as a lower case in Member Card
* Fix Attendance Test

### CODE REVIEW

File: src/test/java/seedu/ccacommander/model/attendance/AttendanceTest.java [hunk 1]
```diff
@@ -3,50 +3,49 @@
...
 
 import org.junit.jupiter.api.Test;
 
-import seedu.ccacommander.model.event.Event;
 import seedu.ccacommander.testutil.AttendanceBuilder;
-import seedu.ccacommander.testutil.EventBuilder;
 
 public class AttendanceTest {
 
     @Test
```
Comment (andrechuakj) [inline src/test/java/seedu/ccacommander/model/attendance/AttendanceTest.java:21]: Thanks for correcting this

File: src/test/java/seedu/ccacommander/commons/util/StringUtilTest.java [hunk 2]
```diff
@@ -141,23 +141,27 @@ public void getDetails_nullGiven_throwsNullPointerException() {
...
 
     @Test
-    public void capitaliseString_blankInput_correctResult() {
-        assertEquals("", StringUtil.capitaliseString(""));
-        assertEquals(" ", StringUtil.capitaliseString(" "));
-        assertEquals("     ", StringUtil.capitaliseString("     "));
+    public void capitaliseWordInString_blankInput_correctResult() {
+        assertEquals("", StringUtil.capitaliseWordsInString(""));
+        assertEquals("", StringUtil.capitaliseWordsInString(" "));
+        assertEquals("", StringUtil.capitaliseWordsInString("     "));
```
Comment (andrechuakj) [inline src/test/java/seedu/ccacommander/commons/util/StringUtilTest.java]: Maybe we can add a test where the string starts with white space, and another test for a string where there are multiple white spaces between words?

**Review by andrechuakj — COMMENTED** [review]
LGTM overall, left a comment about testing of capitaliseWordsInString

File: src/test/java/seedu/ccacommander/model/attendance/AttendanceTest.java (hunk 1 above)
Comment (wr1159) [reply src/test/java/seedu/ccacommander/model/attendance/AttendanceTest.java:21]: No problem 👍

File: src/test/java/seedu/ccacommander/commons/util/StringUtilTest.java (hunk 2 above)
Comment (wr1159) [reply src/test/java/seedu/ccacommander/commons/util/StringUtilTest.java]: Ah good thinking, I have updated the function and added the tests here https://github.com/AY2324S1-CS2103T-F11-1/tp/pull/150/commits/2d1719834396458ab35ad04aa2b0bbb94b272c99.

**Review by andrechuakj — APPROVED** [review]
LGTM

**Review by ph-nathan — APPROVED** [review]
LGTM

**Pull request merged**